### ISSUE TITLE
Writing of CompositeVideoClip hangs/slows to a crawl in drawing.py

### ISSUE THREAD

**Issue by zamirkhan** [Zulko/moviepy#1150]
We use moviepy extensively, writing thousands of clips. I recently ran into one video which caused our system to hang. There is no obvious corruption to the file, but when it is part of a CompositeVideoClip, the writing of that clip never finishes and whenever I kill the process, below is where the stack trace is executing at:

```
 File "/usr/local/lib/python3.7/site-packages/moviepy/Clip.py", line 95, in get_frame
    return self.make_frame(t)
  File "/usr/local/lib/python3.7/site-packages/moviepy/video/compositing/CompositeVideoClip.py", line 118, in make_frame
    f = c.blit_on(f, t)
  File "/usr/local/lib/python3.7/site-packages/moviepy/video/VideoClip.py", line 586, in blit_on
    return blit(img, picture, pos, mask=mask, ismask=self.ismask)
  File "/usr/local/lib/python3.7/site-packages/moviepy/video/tools/drawing.py", line 45, in blit
    1.0 * mask * blitted + (1.0 - mask) * blit_region)
```
-->

I know this video is partly at fault, but any idea what might be going on and how this could be prevented?

**Comment by tburrows13** [comment]
It seems that CompositeVideoClips have been causing quite a few performance problems lately. I see that you've found #1145 which contains a fix that you could try yourself (which stems from the issues in #1143).

**Comment by keikoro** [comment]
I think this got fixed in #1342. Feel free to reopen if I'm mistaken.
